test/unit/user_test.rb

```diff
@@ -83,4 +83,18 @@ class UserTest < ActiveSupport::TestCase
     node2 = rusers(:lurker).node.find_by_nid(20)
      assert_equal [node2], lurker.content_followed_in_past_period(2.hours.ago)
   end
+
+  test 'returns nodes coauthored by user with coauthored_notes method' do
+    jeff = rusers(:jeff)
+    bob = rusers(:bob)
+    assert bob.coauthored_notes.empty?
+
+    jeffs_note = node(:one)
+    jeffs_note.tag.create(name: "with:bob")
```

Comment:
See this example:

https://github.com/publiclab/plots2/blob/a91d7faf2b74e928c1013e1bcd1f4816c5cbf9e3/test/integration/node_insert_extras_test.rb#L21